src/main/java/app/linguistai/bmvp/service/profile/HobbyLoader.java

```diff
@@ -58,9 +60,11 @@ public void run(ApplicationArguments args) throws Exception {
         
         try {
             hobbyRepository.saveAll(Arrays.asList(hobbies));
+            log.info("Hobbies loaded into the database successfully.");
         } catch (DataIntegrityViolationException e) {
-            System.out.println("Hobbies already exist");
```

Comment:
The end of an era

Reply:
Maybe the real treasure was the hobbies we confirmed the existence of along the way